ClosedXML_Tests/Excel/Rows/RowTests.cs

```diff
@@ -263,5 +259,14 @@ public void NegativeRowNumberIsInvalid()
 
             Assert.IsFalse(row.RangeAddress.IsValid);
         }
+
+        [Test]
+        public void DeleteRowOnWorksheetWithComment()
+        {
+            var ws = new XLWorkbook().AddWorksheet();
+            ws.Cell(4, 1).Comment.AddText("test");
+            ws.Column(1).Width = 100;
+            ws.Row(1).Delete();
```

Comment:
The test does not have as `Assert`. If it just aims to check that the code does not fail I'd prefer to make it explicit. E.g. `Assert.DoesNotThrow(() => ws.Row(1).Delete());`.

Reply:
Of course. I forget that NUnit has this assertion.